contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -1,68 +1,704 @@
         exists: optional
+        doc: |
+          Current of low-energy electrons to the sample for charge neutralization.
+        flood_gun(NXactuator):
+          doc: |
+            Flood gun creating a current of low-energy electrons.
+            
+            This should be a link to /entry/instrument/flood_gun.
+    data(NXdata_mpes):
+      doc: |
+        The default NXdata field containing a view on the measured data.
+        This NXdata field contains a collection of the main relevant fields (axes).
+        If you want to provide additional views on your data, you can additionally use
+        the generic NXdata group of NXentry.
+        
+        The naming of fields should follow the convention defined in the
+        :ref:`NXdata_mpes` base class:
+        
+        - **energy**: Calibrated energy (kinetic or binding energy). Unit category: NX_ENERGY (e.g., eV).
+        - **momentum**: Calibrated momentum axis. Unit category: NX_ANY (e.g., kg*m/s).
+        - **kx**: Calibrated x axis in k-space. Unit category: NX_ANY (e.g., 1/Angström).
+        - **ky**: Calibrated y axis in k-space. Unit category: NX_ANY (1/Angström).
+        - **kz**: Calibrated z axis in k-space. Unit category: NX_ANY (1/Angström).
+        - **angular0**: Fast-axis angular coordinate (or second slow axis if angularly integrated).
+          Unit category: NX_ANGLE
+        - **angular1**: Slow-axis angular coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
+          Unit category: NX_ANGLE
+        - **spatial0**: Fast-axis spatial coordinate (or second slow axis if spatially integrated)
+          Unit category: NX_LENGTH
+        - **spatial1**: Slow-axis spatial coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
+          Unit category: NX_LENGTH
+        - **delay**: Calibrated delay time. Unit category: NX_TIME (s).
+        - **polarization_angle**: Linear polarization angle of the incoming or
+          outgoing beam. This could be a link to
+          /entry/instrument/beam/incident_polarization_angle or
+          /entry/instrument/beam/final_polarization_angle if they exist.
+          Unit category: NX_ANGLE (° or rad)
+        - **ellipticity**: Ellipticity of the incoming or outgoing beam.
+          Could be a link to /entry/instrument/beam/incident_ellipticity or
+          /entry/instrument/beam/final_ellipticity if they exist.
+          Unit category: NX_ANGLE (° or rad)
+      \@signal:
+        enumeration: [data]
+      data(NX_NUMBER):
+        unit: NX_ANY
+        doc: |
+          Represents a measure of one- or more-dimensional photoemission counts, where the
+          varied axis may be for example energy, momentum, spatial coordinate, pump-probe
+          delay, spin index, temperature, etc. The axes traces should be linked to the
+          actual encoder position in NXinstrument or calibrated axes in NXprocess.
+      energy(NX_NUMBER):
```

Comment:
This is defined in the NXmpes_data, no? Let's remove it here then.

Reply:
Removed, now it just says:
```
energy(NX_NUMBER):
  \@type:
    exists: recommended
```